index.js

```diff
@@ -4,7 +4,13 @@
 var { requireUp } = require('./requireUp');
 var { DEFAULT_EXTENSIONS } = require('./constants');
 
-var rules = requireUp('eslint-local-rules', DEFAULT_EXTENSIONS, __dirname);
+// First check for local rules in the current working directory and its ancestors (enables npm/yarn workspaces support)
```

Comment:
How about including pnpm as well?? 
I use this plugin in pnpm workspaces.

```suggestion
// First check for local rules in the current working directory and its ancestors (enables npm/yarn/pnpm workspaces support)
```

Reply:
Done!